text/000-2018-2019-roadmap.md

```diff
@@ -0,0 +1,88 @@
+4. Deliberations among the Ember core teams.
+
+The goal of the RFC is to align the Ember community around a set of shared, achievable goals that balance the needs of existing users with the need to grow and support new users.
+
+## Detailed design
+
+Now that all of Ember Octane’s features are code complete, it’s time to turn our attention to what we’ll focus on over the next year.
+
+In this roadmap, our goal is to build on last year’s momentum and capitalize on the cutting-edge foundation we laid with Ember Octane.
+
+- **Continue simplifying Ember.** With many new APIs introduced in Octane, our focus this year should be removing features that feel out of date. The overall API surface area of Ember will shrink substantially, making it smaller, faster, and more approachable to new learners.
+- **Modernize our build system.** This year, we will put the finishing touches on Embroider and make it the default build pipeline for new applications, unlocking optimizations like tree-shaking and route-based code splitting.
+- **Better a11y by default**. We will fix the router to work out-of-the-box with assistive technologies, and give developers more tools for understanding and improving the accessibility of their Ember applications.
+- **Optimize for growth.** A feeling of momentum is critical for keeping the Ember community healthy. We will double down on evangelizing Ember Octane and making Ember approachable to a new generation of web developers.
+
+### Continue simplifying Ember
+
+Ember Octane put us on a strong footing, modernizing the most foundational APIs in Ember. We will build on that foundation to dramatically simplify the framework, **reducing the overall number of concepts** so that Ember is easier to learn, use, and debug.
+
+We will **extract or deprecate functionality that is no longer widely needed**. With `get()` and `set()` mostly gone, so goes the need for many Ember-specific utility functions. Some utilities can be moved into optional packages for those who need them, while others may be better replaced by libraries like [Lodash](https://lodash.com/).
+
+Candidates for deprecation or extraction include:
+
+- **Object utilities** such as `isBlank`, `compare`, `trySet`, etc.
+- **String utilities** such as `camelize` and `dasherize`.
+- **Computed property macros** such as `alias`, `mapBy`, `notEmpty`, etc.
+- **Legacy object model features** such as observers, `setProperties`, `incrementProperty`, etc.
+
+Perhaps the longest-awaited simplification is **eliminating controllers** from the Ember programming model, which we will do this year. This will require us to find a new home for query parameters.
+
+Building on the simplicity of Glimmer components, we’ll also implement **template imports**, which make it explicit which helper or component is being invoked, and **named blocks**, which make it even easier to build ergonomic, reusable components.
```

Comment:
Where they exist, can we link to the RFCs for features like template imports throughout this document? I imagine that people will find it very useful to know what has already been planned, particularly if they are not already aware what these terms mean in a practical sense. That will help people reviewing the RFC better understand how the planned direction will manifest!